developer/frameworks/articles/09-info-framework/01-intro.markdown

```diff
@@ -0,0 +1,17 @@
+# Introduction to Content Sets
+
+In previous versions of the Liferay, the Asset Publisher was your most powerful tool for delivering content to users. With the Asset Publisher, administrators define either static lists of assets to display to a user or dynamic lists based on criteria like tags, categories, or asset type. In @product-ver@ Content Sets take the core idea of defining different types of asset lists and expands it. Content Lists are created outside of the context of a specific application or widget and can be used and re-used across different channels and applications.
```

Comment:
Let's not start with Asset Publisher. That might be ok for old time Liferay users, but for new users, we need to be able to explain what Content Sets are without relying on it.

Reply:
Also, let's not use the term "Asset" for this, since it means a different thing in the industry (it usually refers to an image, video, etc.). We have kept it in "Asset Publisher" for historic reasons, but for anything new we are using the more general term "Content" on purpose.

Reply:
What I would like to see in this intro is how modern sites use lists to show the right content to the right users thorought a site and also in other channels. Then you can continue with how Content Sets allow defining lists independently from where it's going to be displayed, facilitating reuse across pages or even devices (thanks to the provided API). End mentioning personalization, allowing different users to see the most appropriate content for their needs.